proto/v1/places.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option go_package = "./proto";
+package datacommons.v1;
+
+import "place.proto";
+
+message PlaceInfoRequest {
+  string entity = 1;
+}
+  
+message PlaceInfoResponse {
+  string entity = 1;
+  datacommons.PlaceMetadata data = 2;
```

Comment:
data -> info? In the response, there is top level "data" and the data here. I feel it's helpful to have more concrete field name here.

Reply:
Done.